src/main/java/grupa235/proiectColectiv/services/impl/MovieServiceImpl.java

```diff
@@ -23,4 +23,29 @@
     public Optional<Movie> findById(Integer id){
         return movieRepository.findById(id);
     }
+
+    @Override
+    public List<Movie> FilterMoviesByGenres(String genre) {
+        return movieRepository.findAll();
+    }
+
+    @Override
+    public List<Movie> SortMoviesByAddedDate() {
```

Comment:
nu are niciun sens sa apelezi doar findAll aici. poti folosi findAll si dupa sa sortezi tot in Service